### PR TITLE
[CTW-401] Add way to check library version for component library

### PR DESCRIPTION
**Pull request by azambrzycki-htd**
[CTW-401](https://zeushealth.atlassian.net/browse/CTW-401)

- Added library version to window global variable
- Exported library version so customer can use it in their own debug view

Screenshot 2022-10-20 at 13 13 21

### CODE REVIEW

**Review by WesleyKapow — COMMENTED** [review]
Looks like a pretty simple solution. I've got two thoughts:
1. I don't think we need to export it as they'll also have access to the global AND to their own package.json.
2. Wondering if there's a better spot for this. Seems weird to have it in `index.ts` as that is typically just to list exports. Maybe it belongs in `CTWProvider`?

File: .changeset/thin-tomatoes-pump.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@zus-health/ctw-component-library": minor
+---
+
+Added library version to window.CTWComponentLibrary.version variable and version is now exported to dependants
```
[the remark this answers is not in the document]
Comment (pmadria3) [reply .changeset/thin-tomatoes-pump.md:5]: And if we remove export then rephrase this as needed

**Comment by azambrzycki-htd** [comment]
> 1. I don't think we need to export it as they'll also have access to the global AND to their own package.json.
Using global is not good idea because like in remix some frontends can be rendered server side where there is no window global. In package.json you don't have exact version number ("~", "^" in version).

> 2. Wondering if there's a better spot for this. Seems weird to have it in `index.ts` as that is typically just to list exports. Maybe it belongs in `CTWProvider`?
I moved it in new commit

**Review by WesleyKapow — APPROVED** [review]
LGTM
